build.sbt

```diff
@@ -65,11 +65,11 @@ lazy val `quill-jdbc` =
       fork in Test := true,
       libraryDependencies ++= Seq(
         "com.zaxxer"              % "HikariCP"             % "2.6.1",
-        "mysql"                   % "mysql-connector-java" % "5.1.42"     % Test,
-        "com.h2database"          % "h2"                   % "1.4.195"    % Test,
-        "org.postgresql"          % "postgresql"           % "42.1.1"     % Test,
-        "org.xerial"              % "sqlite-jdbc"          % "3.8.11.2"   % Test,
-        "com.microsoft.sqlserver" % "mssql-jdbc"           % "6.1.0.jre8" % Test
+        "mysql"                   % "mysql-connector-java" % "5.1.42"             % Test,
+        "com.h2database"          % "h2"                   % "1.4.195"            % Test,
+        "org.postgresql"          % "postgresql"           % "42.1.1"             % Test,
+        "org.xerial"              % "sqlite-jdbc"          % "3.8.11.2"           % Test,
+        "com.microsoft.sqlserver" % "mssql-jdbc"           % "6.1.7.jre8-preview" % Test
```

Comment:
@fwbrasil It seems that they are tagging bugfix releases as pre-release (or preview). See https://github.com/Microsoft/mssql-jdbc/releases. And the next "stable" release will have 6.2.0 version.